ui/selectors/selectors.js

```diff
@@ -1823,12 +1823,35 @@ export function getNumberOfAllUnapprovedTransactionsAndMessages(state) {
 export const getCurrentNetwork = createDeepEqualSelector(
   getAllNetworks,
   getProviderConfig,
+  /**
+   * Get the current network configuration.
+   *
+   * @param {Record<string, unknown>[]} allNetworks - All network configurations.
+   * @param {Record<string, unknown>} providerConfig - The configuration for the current network's provider.
+   * @returns {{
```

Comment:
I added a JSDoc block so that `tsc` could help spot any instances where this change was breaking.

Specifically the only difference in output before and after is that the `id` property is now optional, whereas before it was guaranteed to exist on the returned object (if an object was returned at all). But luckily nothing was relying on that.